variables.tf

```diff
@@ -14,10 +14,94 @@
  * limitations under the License.
  */
 
-variable "project_id" {
-  description = "The project ID to deploy to"
+/******************************************
+  Required variables
+*******************************************/
+
+variable "organization_id" {
+  description = "GCP Organization ID"
+  type        = string
+}
+
+variable "billing_account" {
+  description = "The ID of the billing account to associate projects with."
+  type        = string
+}
+
+variable "group_org_admins" {
+  description = "Google Group for GCP Organization Administrators"
+  type        = string
+}
+
+variable "group_billing_admins" {
+  description = "Google Group for GCP Billing Administrators"
+  type        = string
+}
+
+variable "default_region" {
+  description = "Default region to create resources where applicable."
+  type        = string
+}
+
+/******************************************
+  Optional variables
+*******************************************/
+
+variable "credentials_file_path" {
```

Comment:
This can be removed: our standard is to defer to the supported credentials environment variables rather than using an input variable.

Reply:
Yep, I realized this is a legacy from a previous version and is no longer required.